packages/fx-core/templates/plugins/resource/hosting/bicep/webapp/provision.template.bicep

```diff
@@ -0,0 +1,20 @@
+// Resources for bot
+module webAppProvision '\{{PluginIdPlaceholder.Provision.webapp.path}}' = {
```

Comment:
we can think about whether we use "app-service" or "webapp" as it alias, since this will lead to different template folder name. CC @hund030